yt/yql/plugin/bridge/interface.h

```diff
@@ -96,8 +96,10 @@ using TFuncBridgeRun = TBridgeQueryResult*(
     const char* impersonationUser,
     const char* queryText,
     const char* settings,
+    int settingsLength,
     const TBridgeQueryFile* files,
-    int fileCount);
+    int fileCount,
+    int executeMode);
```

Comment:
It can't. We have 2 enums:
1. [NYT::NYqlClient::EExecuteMode](https://github.com/ytsaurus/ytsaurus/blob/798da40d10e94e2530e1413978c328b40746a47f/yt/yt/ytlib/yql_client/public.h#L10). Lives in ytsaurus and passed to yql_client as int
2. [NYql::NEmbedded::EExecuteMode](https://github.com/ydb-platform/ydb/blob/9c92253aebf1829c27ce2e7337b635f73d92beb8/ydb/library/yql/public/embedded/yql_embedded.h#L30). Lives in ydb and doesn't have an int representation

Logically we should use a first one, but it can't be used since interface lives in ydb repository

Probably we can use the second one, with some additional work
I see that we use `const char*` as parameters to this interface, and it says "This is a plain C version of an interface described in yt/yql/plugin/plugin.h." in the file, so we may have troubles including 2nd enum